rllib/evaluation/worker_set.py

```diff
@@ -362,6 +362,38 @@ def num_remote_worker_restarts(self) -> int:
         """Total number of times managed remote workers have been restarted."""
         return self.__worker_manager.total_num_restarts()
 
+    @DeveloperAPI
+    def sync_connectors(self) -> None:
+        """Synchronizes the connectors of this WorkerSet's EnvRunners.
```

Comment:
A `no-sync` flag in the connectors might be helpful in case a connector can be synched (e.g. running mean), but should not (e.g. MA settings).

Reply:
Good question. If for example in an MA setting with highly heterogenous observations agents move in some very different regions, it might stabilize training of their common modules, if they demean or normalize observations with their decentralized means and stds. But for sake of ease, let's keep it like it is. This can be tested and changed in a future PR.